packages/react/src/hooks/use-last-factor.ts

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useMemo, useCallback } from "react";
+import { isBrowser } from "@slashid/react-primitives";
+import { Handle, isFactor } from "../domain/types";
+import { useConfiguration } from "./use-configuration";
+import { useSlashID } from "./use-slash-id";
+import { Factor } from "@slashid/slashid";
+
+export const STORAGE_LAST_FACTOR_KEY = "@slashid/LAST_FACTOR";
```

Comment:
Should this be prefixed with the organization ID? Otherwise if you use different orgs (especially in the hosted login page) you are likely to get an incorrect factor

Reply:
good point!